### ISSUE TITLE
Way to Reset Zitadel Administrator password if locked out

### ISSUE THREAD

**Issue by melgo-tb** [zitadel/zitadel#9112]
### Preflight Checklist

- [x] I could not find a solution in the existing issues, docs, nor discussions
- [x] I have joined the [ZITADEL chat](https://zitadel.com/chat)

### Describe your problem

Currently, there appears to be no straightforward method to reset the Zitadel administrator password if the existing password is unknown. This can lead to scenarios where administrators are entirely locked out of the system, even if they have access to the underlying server or CLI.

This lack of a password recovery or reset mechanism is a critical gap for scenarios involving lost credentials or administrative handovers.

### Describe your ideal solution

As an administrator, I want the ability to reset the Zitadel administrator password from the CLI or server without needing the current password so that I can regain access to the system in case of lockout or forgotten credentials.

### Environment

Self-hosted

### Additional Context

It would be helpful to add a feature that enables password reset directly via the CLI or server access, leveraging authentication mechanisms such as root or superuser privileges. This would ensure that administrators are not permanently locked out while maintaining security and access control integrity.

**Comment by melgo-tb** [comment]
https://github.com/zitadel/zitadel/issues/6466#issuecomment-2536605107 #

**Comment by muhlemmer** [comment]
> This might be helpful
> 
> [#6466 (comment)](https://github.com/zitadel/zitadel/issues/6466#issuecomment-2536605107)

Yes, the way to reset the admin user is over the system API as described in the above comment. This is also how we manage access in our hosted solution on ZITADEL Cloud after we perform proper verification of the persona trying to reset access.

> It would be helpful to add a feature that enables password reset directly via the CLI or server access, leveraging authentication mechanisms such as root or superuser privileges. 

That's exactly what the system API provides.

Other alternatives exist (while the user still has access):

- Make sure you have proper SMTP configured for password resets
- Create a service user with `IAM_OWNER` or `IAM_USER_MANAGER` as store the credentials or private key someplace safe. - Add a second admin human user from within your organization as backup.

These alternatives may also help in case the primary admin becomes physically unavailable. Eg accident, sickness or change of workplace.

As a current mechanism exists, I'm closing this issue.
